### PR TITLE
WA hotfixes New location and tole changes

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-3879
https://tools.hmcts.net/jira/browse/EUI-3860
https://tools.hmcts.net/jira/browse/EUI-3900

### Change description ###

Adding New Location 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/work-allocation/guards/tribunal-caseworker-guard.ts [hunk 1]
```diff
@@ -16,8 +15,7 @@ export class TribunalCaseworkerGuard implements CanActivate {
         const userInfoStr = this.sessionStorageService.getItem('userDetails');
         if (userInfoStr) {
             const userInfo: UserInfo = JSON.parse(userInfoStr);
-            isActivate = userInfo.roles.includes(TribunalCaseworkerGuard.caseWorkerRole) ||
-            userInfo.roles.includes(TribunalCaseworkerGuard.seniorCaseWorkerRole);
+            isActivate = userInfo.roles.includes(TribunalCaseworkerGuard.caseWorkerRole);
```
Comment (rayliangatos) [inline src/work-allocation/guards/tribunal-caseworker-guard.ts:18]: Is there any unit tests(changes of unit test) to cover this?

**Review by connorpgpmcelroy — DISMISSED** [review]
LGTM

File: src/work-allocation/guards/tribunal-caseworker-guard.ts (hunk 1 above)
Comment (udaydenduluri33) [reply src/work-allocation/guards/tribunal-caseworker-guard.ts:18]: @rayliangatos yes please

**Review by connorpgpmcelroy — COMMENTED** [review]
Agree with Ray that if able to write unit tests for this (and they don't already exist) we should

**Review by rayliangatos — DISMISSED** [review]

**Review by leepavlouhmcts — APPROVED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]

**Review by leepavlouhmcts — DISMISSED** [review]

**Review by Mo-Lala84 — APPROVED** [review]
Approved

**Review by leepavlouhmcts — APPROVED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Pull request merged**